unlock-protocol-com/blog/unlock-fiat-subscription.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Unlock Launches Support for Fiat NFT Recurring Subscriptions
+authorName: searchableguy
+publishDate: September 27, 2022
+description: Users can now subscribe to NFTs using fiat payment options such as credit cards.
+image: /images/blog/unlock-fiat-subscription/main.png
+---
+
+![Fiat subscription confirmation page](/images/blog/unlock-fiat-subscription/main.png)
+
+We recently launched support for [crypto based NFT subscriptions](./recurring-subscription-nft.md) which allowed people to pre-approve an amount in erc20 to be charged at a regular interval set in the membership contract.
```

Comment:
@searchableguy this URL throws a 404 in staging. looks like it's linking to the markdown file, not the rendered blog post.